### PR TITLE
gdal 2.0.0

### PR DESCRIPTION
**Pull request by mkrapp**

### CODE REVIEW

File: Library/Formula/gdal.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Gdal < Formula
   desc "GDAL: Geospatial Data Abstraction Library"
   homepage "http://www.gdal.org/"
-  url "http://download.osgeo.org/gdal/1.11.2/gdal-1.11.2.tar.gz"
-  sha256 "66bc8192d24e314a66ed69285186d46e6999beb44fc97eeb9c76d82a117c0845"
+  url "http://download.osgeo.org/gdal/2.0.0/gdal-2.0.0.tar.gz"
+  sha256 "53761563ff53c5bf27bff7c4d6cab8bb1634baccefda05348e0f3b7acaf4c9e6"
   revision 2
```
Comment (mikemcquaid) [inline Library/Formula/gdal.rb]: This `revision` will need to be removed as it's a version update, sorry!

Comment (mkrapp) [inline Library/Formula/gdal.rb]: Remove the whole line?

Comment (mikemcquaid) [inline Library/Formula/gdal.rb]: Yup, thanks!

**Comment by mkrapp** [comment]
I've updated to 2.0.0 because I also had the same issue #42031.

**Comment by mkrapp** [comment]
From the tests, I see that `brew test --verbose points2grid` fails because the `gdal` library cannot be found `/usr/local/lib/libgdal.1.dylib`. The link to the library now being `/usr/local/lib/libgdal.20.dylib`.

Setting `ln -s /usr/local/lib/libgdal.dylib /usr/local/lib/libgdal.1.dylib` and the `points2grid` test passes.

Any idea how to fix that?

**Comment by mikemcquaid** [comment]
@mkrapp everything output by `brew uses gdal` probably needs a `revision 1` line added (or that number incremented if the line is already there.

**Comment by DomT4** [comment]
Yeah, when we looked at this before it was a revision party situation. If there's a ton of `gdal` stuff in other Homebrew core taps it's probably courteous to give them a heads up on this as well.

**Comment by mkrapp** [comment]
@mikemcquaid should I check and go for it? Or don't we upgrade to 2.0.0?

**Comment by DomT4** [comment]
Add or update the revision lines on each of these formulae. Use a unique commit (In the same PR) for each updated formula with the commit message along the lines of `formula_name: revision for gdal`, Thanks!

```
gdal-grass
grass
liblas
mapnik
mapserver
open-scene-graph
orfeo
pdal
points2grid
postgis
simple-tiles
```

**Comment by mkrapp** [comment]
I start with the 3 formulas from the tests (`pdal`, `points2grid`, and `simple-tiles`) to see if that works then, `simple-tiles` still fails with 

    ERROR 1: Stub implementation in GDAL 2.0
    ERROR 1: Stub implementation in GDAL 2.0

**Comment by dunn** [comment]
Tap formulae that may need revisioning:
```
homebrew/science/gmt
homebrew/science/gmt4
homebrew/science/osgearth
homebrew/science/qgis
homebrew/science/sumo
homebrew/versions/postgis20
homebrew/x11/xastir
```

**Comment by mkrapp** [comment]
@dunn Could we create another pull request for taps and just update formulae in `homebrew`?

**Comment by bfontaine** [comment]
@mkrapp You’ll have to make one PR per affected tap.

**Comment by mkrapp** [comment]
@bfontaine I'm so looking forward to this. BTW, how can I test if the new version of `gdal` does not break any other formula? And the revision flag just indicates that some of the dependencies have been updated, right?

**Comment by bfontaine** [comment]
> how can I test if the new version of gdal does not break any other formula?

Install the new version of  `gdal` locally, then install formulae that depend on it.

> And the `revision` flag (or increment) just indicates that some of the dependencies have been updated, right?

It forces a reinstall of the formula, so it’ll be built against the new `gdal` version.

**Comment by mkrapp** [comment]
I tried to install the dependent formulae, some failed:

`gdal-grass` passes with patch
`grass` passes
`liblas` passes
`mapnik` passes
`mapserver` fails
`open-scene-graph` OK with update to `3.4.0`; fails for `3.3.3`
`orfeo` fails 
`pdal` passes
`points2grid` passes
`postgis` passes
`simple-tiles` passes; test still fails with `ERROR 1: Stub implementation in GDAL 2.0` (see above)

There is a lot going wrong, are you sure all those packages worked for `gdal` < `2.0.0`? Just asking...

What's you're advise on how to proceed?

**Comment by mkrapp** [comment]
Here's a quick update on my latest commits:

* updated `orfeo` to `5.0.0` (needs `ossim`, which I copied, modified from [here](https://github.com/imintel/homebrew-imintel/blob/master/ossim.rb) and added to my repo)
* updated `mapserver` to `7.0.0` (builds with `cmake`)

All packages pass compilation on my machine.

**Comment by mkrapp** [comment]
Is there a way to increase the timeout of the build bot?

`Build timed out (after 65 minutes)`

**Comment by mikemcquaid** [comment]
We can but fixing `postgis` and `ossim` is probably worth doing before we bump the timeout.

**Comment by DomT4** [comment]
If we could get comments on any patches used as well, with upstream bug submissions/PRs/commits etc that'd be great. One commit per changed formula would be a lot easier to keep on top of, if at all possible.

**Comment by mkrapp** [comment]
@mikemcquaid agreed!

 Installing `postgis` on my machine (yosemite) works well, both

`brew install --verbose --build-bottle postgis`

and 

`brew install postgis --verbose`.

The only error related to `postgis` I find on Jenkins' console output is 

```
syslog -F '$((Time)(local)) $(Sender)[$(PID)]: $Message' -k Time ge 1440081439 -k Sender kernel -o -k Time ge 1440081439 -k Sender sandboxd | grep deny > /Users/brew/Jenkins/prs/version/yosemite/logs/postgis/sandbox.build.log 
Error: Operation not permitted - (stage/usr/local/Cellar/postgresql/9.4.4/lib/postgis-2.1.so, /usr/local/Cellar/postgresql/9.4.4/lib/postgis-2.1.so)
```

Now checking `ossim`, seems like a missing `GEOS` installation.

**Comment by mkrapp** [comment]
@DomT4 I had to patch `ogrgrass.h` for `gdal-grass` because of a wrong type that gave an error. Probably needs a fix in the `gdal-grass` source (for which I haven't submitted a bug report, yet):

`GetFeatureCount( int )` in `ogrgrass.h` returns `int` but should return `GIntBig` instead. Error message when compiling the source (`gdal-grass-1.4.3.tar.gz`) is:

```
In file included from ogrgrassdriver.cpp:30:
./ogrgrass.h:54:25: error: virtual function 'GetFeatureCount' has a different return type ('int') than the function it overrides
      (which has return type 'GIntBig' (aka 'long long'))
    int                 GetFeatureCount( int );
    ~~~                 ^

```

Regarding the commits, I'm pretty sure I commit one change at a time.

**Comment by mkrapp** [comment]
That build failed and I don't know why? What does `ref HEAD is not a symbolic ref` mean?
```
Error Message

failed: brew pull --clean https://github.com/Homebrew/homebrew/pull/42845
Stacktrace

        fatal: ref HEAD is not a symbolic ref
==> Applying patch
Applying: update to gdal 2.0.0
Using index info to reconstruct a base tree...
M	Library/Formula/gdal.rb
Falling back to patching base and 3-way merge...
Auto-merging Library/Formula/gdal.rb
CONFLICT (content): Merge conflict in Library/Formula/gdal.rb
Failed to merge in the changes.
Patch failed at 0001 update to gdal 2.0.0
The copy of the patch that failed is found in:
   /usr/local/.git/rebase-apply/patch
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".
Error: Patch failed to apply: aborted.
```

**Comment by mikemcquaid** [comment]
It looks like there may be a merge conflict.

**Comment by mikemcquaid** [comment]
But I've retried the build to make sure.

**Comment by DomT4** [comment]
>  (for which I haven't submitted a bug report, yet):

Please do so and add that bug report above the patch in formula, Thanks!

> It looks like there may be a merge conflict.

Apparently so. This needs to be rebased on an up-to-date master branch before our CI can look over it again.

**Comment by mkrapp** [comment]
@DomT4 done!

**Comment by mkrapp** [comment]
BTW with `orfeo` updated to 5.0.0 this PR should be referenced: #41363

**Comment by mkrapp** [comment]
If somebody can get back to this, I'm still confused what to do about the merge conflict and how to solve it (for CI).

**Comment by bfontaine** [comment]
@mkrapp I guess you can cancel your merge commit in e32812d (`git reset --hard HEAD^`) then rebase on top of the master branch (`git rebase master`) and force-push again (`git push -f`)

**Comment by mkrapp** [comment]
Am I too stupid? Probably, but anyhow. Any advise is appreciated to circumvent this "merge" issue.

**Comment by mikemcquaid** [comment]
Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

**Comment by elgertam** [comment]
I saw this was open and was wondering the status. Is this something I can help with?

**Comment by mkrapp** [comment]
indeed, some help would be appreciated @elgertam. I'm struggling to understand the multitude iof (error/FAILED) build messages.

**Comment by elgertam** [comment]
Understood, @mkrapp. I'll start by attempting to build on my machine. This is my first attempt contributing to Homebrew, so bear with me.

**Comment by MatzFan** [comment]
Happy to help with testing & get the gdal 2.0 formula out. If this may be useful please let me have build/test instructions. Am currently running OS X 10.9.5

**Comment by dunn** [comment]
Oop, now this'll need to be rebased.

**Comment by mkrapp** [comment]
Rebase done! Now, let's see...

File: Library/Formula/gdal.rb [hunk 2]
```diff
@@ -79,11 +79,14 @@ class Gdal < Formula
     depends_on "ant" => :build
     depends_on "swig" => :build
   end
+<<<<<<< HEAD
```
Comment (dunn) [inline Library/Formula/gdal.rb]: You forgot to resolve a conflict here :see_no_evil:

**Comment by dunn** [comment]
This will probably hit the timeout again, so wait to push a fix until we can schedule a build.

**Comment by mkrapp** [comment]
@dunn. Thanks I fixed that and rebased. How do I know when to push a commit?

**Comment by dunn** [comment]
@mkrapp The queue is currently empty so if you're online, push now.

**Comment by dunn** [comment]
We've got test results from 4 hours ago, actually: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/34175/#showFailuresLink

**Comment by DomT4** [comment]
On `postgis` you'll need to see/follow the notes in #41962 to get that to pass.

**Comment by mkrapp** [comment]
Many installs seem to fail because of `open-scene-graph`. I couldn't build it on El Capitan.

**Comment by elgertam** [comment]
@mkrapp Which branch are you working out of, mkrapp or master?

**Comment by mkrapp** [comment]
Hi everybody,

this PR has grown more complex over time and I was wondering if we could break it down somehow to resolve the issues with failed CI's. Here are some comments:

* Is it possible, for example, to test the updates in a kind of "sandbox" environment?
* Can some packages other than `gdal` be updated separately? For example, update `map-server` to `7.0.0` with `gdal 1.11.3`, or add `ossim`, then update `orfeo` and so forth. Same for some of the other packages.

**Comment by elgertam** [comment]
@mkrapp +1

I tested your changes last night, and I'm inclined to agree. I tested last night, and GDAL built and installed just fine. I had some problems getting the gdal-grass formula to install because grass was failing to install, which apparently isn't uncommon.

**Comment by DomT4** [comment]
`ossim` can become its own PR after this is merged. We should try and get `postgis` fixed outside this PR before moving forwards on this too much further as the same set of errors related to that are causing a lot of the failures. The OSG failure is possibly related to #44524.

Sorting out B & C on the above list before we push ahead on this should fix almost all the errors you're seeing, theoretically.

**Comment by bfontaine** [comment]
Ping?

**Comment by elgertam** [comment]
@bfontaine and @DomT4: What else is there to be done here in order to accept the PR? I'm able to build GDAL using @mkrapp's PR, and the only errors I have seen are unrelated to GDAL itself.

**Comment by bfontaine** [comment]
@elgertam The errors you see are formulae that depend on GDAL; we need to fix all of them before merging this PR.

**Comment by mkrapp** [comment]
Any idea how to move forward from here?

**Comment by bfontaine** [comment]
This branch needs to be rebased on top of `master`.

**Comment by mkrapp** [comment]
Hi. Here is a summary of the most recent build failure(s) and what I did to try fixing it. I only checked `10.11.1` (El Capitan)

* `pdal` did not build:
```
dyld: Library not loaded: /usr/local/opt/gdal/lib/libgdal.1.dylib
  Referenced from: /usr/local/Cellar/pdal/0.9.9_1/bin/pdal
  Reason: image not found
Error: pdal: failed
Failed executing: /usr/local/Cellar/pdal/0.9.9_1/bin/pdal info
```
* `points2grid` did not build:
```
dyld: Library not loaded: /usr/local/opt/gdal/lib/libgdal.1.dylib
  Referenced from: /usr/local/Cellar/points2grid/1.3.0_2/bin/points2grid
  Reason: image not found
Error: points2grid: failed
Failed executing: /usr/local/Cellar/points2grid/1.3.0_2/bin/points2grid -i /usr/local/Cellar/points2grid/1.3.0_2/libexec/example.las -o example --max --output_format grid
```
:bulb: However, I'm able to fix the above two errors manually by creating a symbolic link in `/usr/local/opt/gdal/lib`
```
ln -s libgdal.dylib libgdal.1.dylib
```
* `open-scene-graph` did not build:
```
Error: open-scene-graph 3.4.0 did not build
```
:bulb: If I install it manually everything works fine with `cmake` options taken from output of `brew install open-scene-graph --with-gdal --verbose`:
```
wget http://trac.openscenegraph.org/downloads/developer_releases/OpenSceneGraph-3.4.0.zip
unzip OpenSceneGraph-3.4.0.zip
cd OpenSceneGraph-3.4.0
mkdir build && cd build
cmake .. -DCMAKE_C_FLAGS_RELEASE= -DCMAKE_CXX_FLAGS_RELEASE= -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/open-scene-graph/3.4.0 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DBUILD_DOCUMENTATION=OFF -DCMAKE_OSX_ARCHITECTURES=x86_64 -DOSG_DEFAULT_IMAGE_PLUGIN_FOR_OSX=imageio -DOSG_WINDOWING_SYSTEM=Cocoa
make && make install
```
A while ago I also commented on a referenced issue #44524.
* `grass` did not build:
Seems like `wxpython` causes some issues
```
Error: wxpython 3.0.2.0 did not build
```
Issues with `wxpython` are #45381 and #45024. :bulb: However, I was able to install it locally.
* `gdal-grass` did not build because of `wxpython`
* `ossim` did not build because of `open-scene-graph`
* `orfeo` did not build because of `open-scene-graph`

So we can break it down to `open-scene-graph` and `wxpython` (and some missing links of the `gdal` library).
:question: Any ideas how to resolve that?

**Comment by mikemcquaid** [comment]
> pdal did not build:
> points2grid did not build:

These two need a `revision` bump rather than symlink. Not sure about the others.

**Comment by mkrapp** [comment]
OK, `pdal` and `points2grid` are no longer an issue.

But `ossim` caused trouble, which I fixed with an updated version (`1.8.20-1`).

Also, while checking errors in `open-scene-graph` I decided to switch to the latest version (`3.5.1`). I cleaned up the `open-scene-graph` formula but still some errors persist. One is related to a c++11-narrowing error, see #46372, the other is an error in `open-scene-graph` itself: 
```
OpenSceneGraph-3.5.1/include/osgViewer/View:130:91: error: member reference base type
      'const osg::ref_ptr<T> *' is not a structure or union
        template<class T> void setImagePager(const osg::ref_ptr<T>* ip) { setImagePager(ip.get()); }
                                                                                        ~~^~~~
1 error generated.
```
A patched the corresponding file and sent a bug report to the osg-users mailing list.

That formula now installs on my machine (`10.11.1`). Crossing fingers for the build...

**Comment by mkrapp** [comment]
Finally, `open-scene-graph` was built. :joy:

**Comment by Firefishy** [comment]
Latest is gdal 2.0.1: http://download.osgeo.org/gdal/2.0.1/gdal-2.0.1.tar.gz
b55f794768e104a2fd0304eaa61bb8bda3dc7c4e14f2c9d0913baca3e55b83ab

File: Library/Formula/mapserver.rb [hunk 3]
```diff
@@ -29,104 +29,29 @@ class Mapserver < Formula
...
   def install
-    args = [
-      "--prefix=#{prefix}",
-      "--with-proj",
-      "--with-gdal",
-      "--with-ogr",
-      "--with-wfs"
+    args = std_cmake_args
+    args += [
+      "-DCMAKE_BUILD_TYPE=Release",
```
Comment (DomT4) [inline Library/Formula/mapserver.rb]: We pass this already as part of `std_cmake_args`.

File: Library/Formula/mapserver.rb [hunk 4]
```diff
@@ -29,104 +29,29 @@ class Mapserver < Formula
...
-      * Add the following line to php.ini:
-        extension="#{prefix}/php_mapscript.so"
-      * Execute "php -m"
-      * You should see MapScript in the module list
-    EOS
-  end
-
   test do
-    system "#{bin}/mapserver-config", "--version"
+    system "mapserv", "-v"
```
Comment (DomT4) [inline Library/Formula/mapserver.rb]: Please don't remove the `#{bin}` here.

File: Library/Formula/open-scene-graph.rb [hunk 5]
```diff
@@ -33,8 +33,6 @@ class OpenSceneGraph < Formula
   depends_on "qt5" => :optional
   depends_on "qt" => :optional
 
-  # patch necessary to ensure support for gtkglext-quartz
-  # filed as an issue to the developers https://github.com/openscenegraph/osg/issues/34
   patch :DATA
```
Comment (DomT4) [inline Library/Formula/open-scene-graph.rb]: Can you add a link/explanation as to what the new patch is, where it's from, etc? Thanks.

File: Library/Formula/orfeo.rb [hunk 6]
```diff
@@ -1,35 +1,25 @@
 class Orfeo < Formula
   desc "Library of image processing algorithms"
   homepage "http://www.orfeo-toolbox.org/otb/"
-  url "https://downloads.sourceforge.net/project/orfeo-toolbox/OTB/OTB-3.20/OTB-3.20.0.tgz"
-  sha256 "7c405756887842fbdfd6eb99a1fdc3c940817290a99c188414e5af45702d1b9d"
+  url "https://downloads.sourceforge.net/project/orfeo-toolbox/OTB/OTB-5.0.0/OTB-5.0.0.tgz"
+  sha256 "35e2f1d280f4a33b3d38af840a83069b34b2ea99acba18b1a944acccdbf1a976"
+
+  option "with-examples", "Compile and install various examples"
```
Comment (DomT4) [inline Library/Formula/orfeo.rb]: You'll need to deprecate the old options onto the new options here with the:

```ruby
deprecated_option "examples" => "with-examples"
```
style syntax.

File: Library/Formula/orfeo.rb [hunk 7]
```diff
@@ -1,35 +1,25 @@
...
   depends_on :python => :optional
-  depends_on "fltk"
   depends_on "gdal"
-  depends_on "qt"
-
-  option "examples", "Compile and install various examples"
-  option "java", "Enable Java support"
-  option "patented", "Enable patented algorithms"
+  depends_on "open-scene-graph" => ["with-gdal"]
+  depends_on "homebrew/science/insighttoolkit"
```
Comment (DomT4) [inline Library/Formula/orfeo.rb]: Can't have mandatory cross-tap dependencies in the core I'm afraid.

File: Library/Formula/orfeo.rb [hunk 8]
```diff
@@ -1,35 +1,25 @@
...
-    ]
-
-    args << "-DBUILD_EXAMPLES=" + ((build.include? "examples") ? "ON" : "OFF")
-    args << "-DOTB_WRAP_JAVA=" + ((build.include? "java") ? "ON" : "OFF")
-    args << "-DOTB_USE_PATENTED=" + ((build.include? "patented") ? "ON" : "OFF")
-    args << "-DOTB_WRAP_PYTHON=OFF" if build.without? "python"
-
     mkdir "build" do
-      system "cmake", "..", *args
+      system "cmake", ".."
```
Comment (DomT4) [inline Library/Formula/orfeo.rb]: Presumably still want `std_cmake_args` here?

File: Library/Formula/ossim.rb [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+  depends_on "libtiff"
+  depends_on "libgeotiff"
+  depends_on "ffmpeg"
+  depends_on "gdal" => :optional
+  needs :openmp if build.with? "openmp"
+
+  def install
+    args = std_cmake_args
+    args += [
+      "-DCMAKE_BUILD_TYPE=Release",
```
Comment (DomT4) [inline Library/Formula/ossim.rb]: We pass this and the install prefix as part of `std_cmake_args`.

File: Library/Formula/ossim.rb [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+    args += [
+      "-DBUILD_OSSIMPLANET=OFF",
+      "-DBUILD_OSSIMQT4=OFF",
+      "-DBUILD_OSSIMGUI=OFF",
+    ]
+
+    args << "-DBUILD_OSSIMGDAL_PLUGIN=" + (build.with?("gdal") ? "ON" : "OFF")
+    args << "-DBUILD_OSSIMPNG_PLUGIN=" + (build.with?("libpng") ? "ON" : "OFF")
+
+    ossim_dev_home = Dir.pwd
```
Comment (DomT4) [inline Library/Formula/ossim.rb]: Would `Dir.pwd` not be `buildpath` here?

File: Library/Formula/ossim.rb [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+    mkdir "ossim_package_support/cmake/build" do
+      ENV["OSSIM_DEV_HOME"] = ossim_dev_home
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    Dir.mkdir("#{prefix}/etc")
```
Comment (DomT4) [inline Library/Formula/ossim.rb]: Don't need to throw out to `Dir` here. Just `(prefix/"etc").mkpath`

File: Library/Formula/ossim.rb [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+    templatedata = templatedata.gsub(%r{/((\/\/---[^\n]*\n)(\/\/ OSSIM plugin support:\n)(\/\/[^\n]*\n)*(\/\/---[^\n]*\n))/m}, "\\1\nplugin.dir0: $(OSSIM_INSTALL_PREFIX)/lib64/ossim/plugins/\n\n")
+    templatedata = templatedata.gsub(/^(plugin.file)/, '// \1')
+
+    File.open(destination, "w") do |w|
+      w.write(templatedata)
+    end
+  end
+
+  test do
+    system "ossim-info", "--version"
```
Comment (DomT4) [inline Library/Formula/ossim.rb]: Please use the `#{bin}` style links to the executables here.

**Comment by mkrapp** [comment]
@DomT4 Regarding the `orfeo` dependencies. It seems like ITK aka `homebrew/sience/insighttoolkit` is [mandatory](https://www.orfeo-toolbox.org/SoftwareGuide/SoftwareGuidech2.html#x14-220071):

```
CMake Error at build/lib/cmake/OTB-5.0/Modules/OTBITK.cmake:6 (find_package):
  Could not find a package configuration file provided by "ITK" with any of
  the following names:

    ITKConfig.cmake
    itk-config.cmake

  Add the installation prefix of "ITK" to CMAKE_PREFIX_PATH or set "ITK_DIR"
  to a directory containing one of the above files.  If "ITK" provides a
  separate development package or SDK, be sure it has been installed.
```
Also, `homebrew/science/insighttoolkit` needs `homebrew/science/vtk`.

Is it possible to move `orfeo` from the core to `homebrew/science`?

**Comment by dmwelch** [comment]
FYI: `orfeo` is built on top of `ITK` - `ITK` and `VTK` were written by a medical image processing consortium and `orfeo` takes many of their objects as base classes. 🤓

**Comment by nextstopsun** [comment]
Is this ever going to happen? So many people are waiting for gdal 2 to be installed via homebrew, it's really a fundamental spatial library. Sadly, osgeo4mac doesn't have any updates on gdal either. Please don't give up guys, it's definitely worth giving another try.

**Comment by elgertam** [comment]
I concur with @nextstopsun. It appears that, other than a few style and convention issues, we are hung up on the `orfeo` package, which may depend on packages in the `homebrew/science` tap. Can @DomT4 or another homebrew maintainer help us figure out the solution to this: either moving `orfeo` itself to `homebrew/science` or figuring out a way to depend solely on the core packages?

I'm willing to help in any way I can.

@mkrapp: apart from the question about `orfeo`, what else must be done?

**Comment by elgertam** [comment]
For what it's worth, this is what I get when trying to install orfeo with vanilla homebrew:

```
$ brew install orfeo
fltk: This formula either does not compile or function as expected on OS X
versions newer than Yosemite due to an upstream incompatibility.
Error: An unsatisfied requirement failed this build.```

Seems to me that moving it to `homebrew/science` would serve everyone best.

**Comment by mikemcquaid** [comment]
> Seems to me that moving it to homebrew/science would serve everyone best.

:+1:

**Comment by nextstopsun** [comment]
Although orfeo has some really neat image classification functions, dumping it to a science tap seems like a sacrifice for greater good. But what is wrong with fltk?

**Comment by DomT4** [comment]
> But what is wrong with fltk?

As it says on the tin:

> fltk: This formula either does not compile or function as expected on OS X
versions newer than Yosemite due to an upstream incompatibility.

It's already fixed upstream. We're waiting on a new stable release with the fixes included.

**Comment by bfontaine** [comment]
Anything new here?

**Comment by nextstopsun** [comment]
Since new postgresql 9.5 and postgis 2.2.1 are released, gdal 2 is on high demand. Still waiting for it very very much.

**Comment by mkrapp** [comment]
Can somebody take the lead on moving `orfeo` to `home-brew/science`?

**Comment by nextstopsun** [comment]
Any news on this issue?

**Comment by nijikon** [comment]
What is pending here?

**Comment by nextstopsun** [comment]
Still no progress on this issue?

**Comment by mikemcquaid** [comment]
@nijikon The build is failing because someone needs to move `orfeo` to `homebrew/science`, there's a merge conflict and possibly other things.

@nextstopsun No. We'd have posted on this issue if there was. If you would like to create your own pull requests to address the issues above: that'd be great, thanks.

**Comment by oeon** [comment]
There's another pull request for gdal 2+ over at the homebrew-osgeo4mac tap https://github.com/OSGeo/homebrew-osgeo4mac/pull/122 - I thought I would make both camps aware.

**Pull request closed without merging**